ios/engine/KMEI/KeymanEngine.xcodeproj/project.pbxproj

```diff
@@ -1250,6 +1251,23 @@
 			shellPath = /bin/sh;
 			shellScript = "#if which swiftlint >/dev/null; then\n#    swiftlint --config ../../.swiftlint.yml\n#else\n#    echo \"warning: SwiftLint not installed, download from https://github.com/realm/SwiftLint\"\n#fi\n";
 		};
+		CE8D0366240F2FE4002D818C /* ShellScript */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+			);
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
```

Comment:
Is it OK that `xcode-utils.sh` uses this?
```
#!/bin/bash
```

Reply:
Yes that should be fine.